addons/loyalty_account/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Loyalty Account",
+    'summary': "Enables the setup of taxes regarding loyalty.",
+    'description': 'Bridge module between account and loyalty',
+    'category': 'Sales/Sales',
+    'version': '1.0',
+    'depends': ['loyalty', 'account'],
```

Comment:
You can make the core `loyalty` module depend on `account`.
All modules using loyalty depend on `account` already (pos & sale)